book/website/reproducible-research/rdm.md

```diff
@@ -1,3 +1,11 @@
+```{figure} ../../figures/data-ecosystem.jpg
+---
+name: data-ecosystem
+alt: image of the data ecosystem with private and public data
+---
+_The Turing Way_ project illustration by Scriberia. Original version on Zenodo. http://doi.org/10.5281/zenodo.3695300. 
+```
+
```

Comment:
```suggestion

```{figure} ../../figures/data-ecosystem.jpg
---
height: 400px
name: data-ecosystem
alt: image of the data ecosystem with private and public data
---
Open and closed data for reproducibility.
_The Turing Way_ project illustration by Scriberia. Original version on Zenodo. http://doi.org/10.5281/zenodo.3695300. 
```

```